Detector/DetDriftChamber/src/driftchamber/DriftChamber.cpp

```diff
@@ -202,15 +205,15 @@ static dd4hep::Ref_t create_detector(dd4hep::Detector& theDetector,
         //    |                     |
         //    |   F0    F1   F2   F3|
         //    -----------------------
-//     if(layer_id == -1) {
+     if(layer_id == -1) {
```

Comment:
@myliu-hub Is wire ignored now? Plan to update in future?

Reply:
> @myliu-hub Is wire ignored now? Plan to update in future?

Sorry, it was my mistake

Reply:
Thanks. It looks fine now for me.